tests/test_build.py

```diff
@@ -209,3 +209,19 @@ def test_execution_timeout(pages, build_resources, cli):
     )
     assert "Execution Failed" in result.stdout
     assert path_html.joinpath("reports", "loop_unrun.log").exists()
+
+
+def test_build_using_custom_builder(cli, build_resources):
+    """Test building the book template using a custom builder"""
+    books, _ = build_resources
+    config = books.joinpath("config_custombuilder")
+    result = cli.invoke(
+        commands.build(builder="custom", custom_builder="mycustombuilder"),
```

Comment:
@choldgraf @chrisjsewell while writing a test for this PR (adding custom builder passthrough) I am not sure where to specify the `jupyter-book` command line options. The test seems to fail when adding them as keyword arguments to `commands.build`.

Reply:
Ah - I think I have them in the right place now